scripts/ci/_utils.sh

```diff
@@ -712,25 +717,29 @@ function get_local_image_info() {
 function get_remote_image_info() {
     set +e
     # Pull remote manifest image
-    if ! verbose_docker pull "${AIRFLOW_CI_REMOTE_MANIFEST_IMAGE}" >/dev/null 2>&1 ; then
+    if ! verbose_docker pull "${AIRFLOW_CI_REMOTE_MANIFEST_IMAGE}" >/dev/null; then
         echo
         echo "Remote docker registry unreachable"
         echo
         REMOTE_DOCKER_REGISTRY_UNREACHABLE="true"
         return
     fi
+    set -e
+
+    # Docker needs the file passed to --cidfile to not exist, so we can't use mktemp
+    TMP_CONTAINER_ID="remote-airflow-manifest-$$.container_id"
+    trap 'rm -f "$TMP_CONTAINER_ID"' EXIT
```

Comment:
this will override previous trap if you had it set (which is likely), The best solution I found is to wrap the trap code into a function and do it as follows:

```bash
# adding trap to exiting trap
HANDLERS="$( trap -p EXIT | cut -f2 -d \' )"
# shellcheck disable=SC2064
trap "${HANDLERS}${HANDLERS:+;}the_new_function" EXIT
``

Reply:
Oh, you can only have a single trap? How pesky. I'll come up with something then.

Reply:
Fixed by https://github.com/apache/airflow/pull/9105/commits/bce2f6519589355d590af7837000fdf10d11d11b